app/assets/javascripts/application.js

```diff
@@ -351,6 +351,12 @@ $(function() {
   });
 });
 
+function check_for_unavailable_puppetclasses() {
+  if ($('#puppet_klasses #selected_classes .unavailable').size() > 0) {
+    notify('<span>' + __('Some Puppet Classes are unavailable in the selected environment') + '</span>', 'warning')
```

Comment:
I've taken the tab title, that capitalises both, considering that is what the user need to look for when the alert pops up.